utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -263,7 +265,9 @@ Describe 'File/folder tests' -Tag 'Modules' {
                 return
             }
 
-            $defaultsFolder = Get-ChildItem -Directory (Join-Path -Path $moduleFolderPath 'tests' 'e2e') -Filter '*defaults'
+            $defaultsFolder = Get-ChildItem -Directory (Join-Path -Path $moduleFolderPath 'tests' 'e2e') | Where-Object {
+                $_.Name -match '^(defaults)|.*(defaults)$' # the spec BCPRMNFR1 states, that the folder names should start with defaults|waf-aligned. Since it is a should and not a must, need to check for both cases.
```

Comment:
```suggestion
                $_.Name -match '^.*defaults$' # the spec BCPRMNFR1 states, that the folder names should start with defaults|waf-aligned. Since it is a should and not a must, need to check for both cases.
```